baseplate/sidecars/live_data_watcher.py

```diff
@@ -85,7 +85,13 @@ def get_data_to_write(json_data: dict, data: Optional[bytes]) -> Optional[bytes]
         # }
         # If the load type is 'S3', this format is an indication that we support
         # downloading the contents of encrypted files uploaded to S3.
-        if json_data.get("live_data_watcher_load_type") == "S3":
+        is_s3 = False
+        try:
+            is_s3 = json_data["live_data_watcher_load_type"] == "S3"
+        except (KeyError, TypeError):
+            is_s3 = False
+
+        if is_s3:
```

Comment:
While you're making the change, there's a typo on line 107 that I'd like to see fixed `unble` => `unable`